config/runtimes/vllm-multinode-template.yaml

```diff
@@ -203,32 +247,51 @@ objects:
                     echo "$SECONDS seconds elapsed: Still waiting for Global Control Service(GCS) to be ready."
                     echo "For troubleshooting, refer to the FAQ at https://docs.ray.io/en/master/cluster/kubernetes/troubleshooting/troubleshooting.html#kuberay-troubleshootin-guides"
                   fi
-                
+
                   sleep 5
                 done
 
                 export RAY_HEAD_ADDRESS="${HEAD_SVC}.${POD_NAMESPACE}.svc.cluster.local:6379"
                 echo "Attempting to connect to Ray cluster at $RAY_HEAD_ADDRESS ..."
                 ray start --address="${RAY_HEAD_ADDRESS}" --block
             env:
+              - name: RAY_USE_TLS
+                value: '1'
+              - name: RAY_TLS_SERVER_CERT
```

Comment:
why to have 2 envs with the same content?

Reply:
where is the same content?

Reply:
- name: RAY_TLS_CA_CERT
    value: '/etc/ca/tls/tls.crt'
                
- name: RAY_TLS_SERVER_CERT   
  value: '/etc/ray/tls/tls.crt 
  
same content I meant, two different envs with exactly the same value.